charter/index.html

```diff
@@ -238,23 +244,20 @@ <h3>
             The Working Group will deliver the following W3C normative specifications:
           </p>
                 <dl>
-                    <dt id="solid-protocol" class="spec">Solid Protocol v1.0</dt>
+                    <dt id="PUMPKIN-protocol" class="spec">PUMPKIN Protocol v1.0</dt>
                     <dd>
                         <p>
-                            The Solid Protocol specification aims to provide applications with secure and permissioned access to externally stored data in an interoperable way. <q lang="en-GB" cite="https://solidproject.org/TR/2022/protocol-20221231#introduction">An overarching design goal of the Solid ecosystem is to be evolvable and to provide fundamental affordances for decentralised web applications for information exchange in a way that is secure and privacy respecting. In this environment, actors allocate identifiers for their content, shape and store data where they have access to, set access controls, and use preferred applications and services to achieve them.</q>
-                        </p>
-                        <p class="draft-status">
-                          <b>Draft state:</b> Adopted from <a href="https://www.w3.org/groups/cg/solid">W3C Solid Community Group</a>.
+                            The PUMPKIN Protocol specification aims to provide applications with secure and permissioned access to externally stored data in an interoperable way.
                         </p>
-
                         <p>
-                          <b>Adopted Draft</b>: <a href="https://solidproject.org/ED/protocol">Solid Protocol, latest published version (0.10.0) or Editor's Draft</a>
+                            The PUMPKIN Protocol may include protocol details for integration with identity layers and mechanisms; access management and data integrity; notifications about resource changes; and authorization mechanisms.
                         </p>
                         <p>
-                            When possible, the Solid Protocol will evolve while maintaining a high degree of compatibility with existing implementations, of both servers and clients, and with features from prior versions. If incompatible changes have to be made, then they will be done by introducing a stage where both old and new protocols are supported, to allow the implementors to upgrade their systems in a managed way.
+                          <b>Input document</b>: <a href="https://solidproject.org/ED/protocol">Solid Protocol, latest published version (0.10.0) or Editor's Draft</a>, adopted from the <a href="https://www.w3.org/groups/cg/solid">W3C Solid Community Group</a>.
```

Comment:
Putting the following suggestion on the table following https://github.com/solid/solid-wg-charter/pull/69/files#r1533485709 with the understanding that it'd be useful to have on record genuine intentions of the charter proposal:

```suggestion
                          <b>Input documents</b>: <a href="https://solidproject.org/ED/protocol">Solid Protocol, latest published version (0.10.0) or Editor's Draft</a></cite>, adopted from the <a href="https://www.w3.org/groups/cg/solid">W3C Solid Community Group</a>, <a href="https://fcrepo.github.io/fcrepo-specification/">Fedora API Specification 1.0</a>.
```

For those that are unfamiliar with the Fedora API Specification, it is extends LDP and generally compatible with the Solid Protocol. It has a solution for various deliberations that were raised in the Solid Protocol. It also normatively references Web Access Control: https://solid.github.io/web-access-control-spec/ , which is also normatively references in the Solid Protocol. So, this is all *highly* useful and constructive input.

That said, whether the inclusion of this potential input needs to meet a certain criteria, that'd be good to note here, e.g., does it actually need the "approval" of the Fedora Commons/Project to be mentioned in the charter, or can the charter or the WG simply re-use/borrow/incorporate recommendations from the Fedora API Specification 1.0 (CC BY 4.0 licensed) towards "a protocol specification" - I don't see why not but the Solid CG and W3C Team/community can make that call and put it on record.

Reply:
This seems like an unclear formatting:  it says "Solid spec, adopted from SolidCG, Fedora API spec".  I think you mean "Solid spec, adopted from Solid CG; Fedora API spec"?  or bullets, or something.

I think the biggest concern would be provenance of Fedora features and IPR coverage; not that such is impossible to resolve, just needs to be done.

Reply:
For the sake of usability, I'll turn this suggestion into a separate PR.